README.md

```diff
@@ -2,8 +2,9 @@ Fission: Serverless Functions for Kubernetes
 ============================================
 [![Build Status](https://travis-ci.org/fission/fission.svg?branch=master)](https://travis-ci.org/fission/fission)
 [![Go Report Card](https://goreportcard.com/badge/github.com/fission/fission)](https://goreportcard.com/report/github.com/fission/fission)
-[![Fission Slack](http://slack.fission.io/badge.svg)](http://slack.fission.io)
+[![codecov](https://codecov.io/gh/fission/fission/branch/master/graph/badge.svg)](https://codecov.io/gh/fission/fission)
 
```

Comment:
Please move the blank line between slack and fission.io link, so that we can keep icons in one line.